docs/ios/react-native-module.md

```diff
@@ -0,0 +1,100 @@
+# React Native modules
+
+## Creating a new React Native module
+
+A React Native module is a set of standalone React Native views that act as a standalone application.
+
+To create a new React Native module just:
+
+1. Create a new folder inside `app` folder, eg: `app/newModule`, where `newModule` stands for the new module name.
+
+2. Inside `newModule/index.js` export all standalone views.
+
+## Using a React Native module in a playground
+
+In the [`app/native`](https://github.com/kiwicom/mobile/blob/master/app/native.js) import all views from a module of your choice and register them, which then you can use by the defined name, eg:
```

Comment:
Nit: them, which then you can use by the defined name, eg:

Should be: register them by name that you will later refer to from the native side.